ddtrace/contrib/django/apps.py

```diff
@@ -47,19 +47,24 @@ def ready(self):
         )
 
         if settings.AUTO_INSTRUMENT:
-            # trace Django internals
-            insert_exception_middleware()
-            try:
-                patch_db(tracer)
-            except Exception:
-                log.exception('error patching Django database connections')
+            if settings.AUTO_INSTRUMENT_TEMPLATE.get('EXCEPTION_MIDDLEWARE', True):
```

Comment:
when `AUTO_INSTRUMENT` is `True`, just add both middlewares (`TraceMiddleware` and `ExceptionMiddleware`) using two different functions.